Source/Details/UIView+ASConvenience.h

```diff
@@ -51,15 +51,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @protocol ASDisplayNodeViewProperties
 
-@property (nonatomic, assign)           		BOOL clipsToBounds;
-@property (nonatomic, getter=isHidden)  		BOOL hidden;
-@property (nonatomic, assign)           		BOOL autoresizesSubviews;
-@property (nonatomic, assign)           		UIViewAutoresizing autoresizingMask;
-@property (nonatomic, strong, null_resettable)  UIColor *tintColor;
-@property (nonatomic, assign)           		CGFloat alpha;
-@property (nonatomic, assign)           		CGRect bounds;
-@property (nonatomic, assign)           		CGRect frame;   // Only for use with nodes wrapping synchronous views
-@property (nonatomic, assign)           		UIViewContentMode contentMode;
+@property (nonatomic, assign)          BOOL clipsToBounds;
```

Comment:
These were re-indented, because they had actually included tab characters as well as spaces (making it impossible to line up on GitHub even though it did in Xcode)